lib/load.go

```diff
@@ -13,45 +13,43 @@ import (
 	"github.com/qri-io/qri/fsi"
 )
 
+type datasetLoader struct {
+	inst   *Instance
+	source string
+}
```

Comment:
Moving this to it's own struct is lovely. I can see a future PR that moves loading into it's own package, pulling the `noHistoryLoader`, `base.NewLocalDatasetLoader` and `transform.Loader` into one place.